Python/pywarpx/picmi.py

```diff
@@ -463,6 +463,57 @@ def initialize_inputs(self):
             pywarpx.amr.max_level = 0
 
 
+class Cartesian1DGrid(picmistandard.PICMI_Cartesian1DGrid):
+    def init(self, kw):
+        self.max_grid_size = kw.pop('warpx_max_grid_size', 32)
+        self.max_grid_size_x = kw.pop('warpx_max_grid_size_x', None)
+        self.blocking_factor = kw.pop('warpx_blocking_factor', None)
+        self.blocking_factor_x = kw.pop('warpx_blocking_factor_x', None)
+
+        self.potential_xmin = None
+        self.potential_xmax = None
+        self.potential_ymin = None
+        self.potential_ymax = None
+        self.potential_zmin = kw.pop('warpx_potential_lo_z', None)
+        self.potential_zmax = kw.pop('warpx_potential_hi_z', None)
```

Comment:
On lines 405 & 406 of `ElectrostaticSolver.cpp` the `potential_xlo/hi` values are used to set the 0 index values of `phi_bc_values_lo/hi` which is used to set the boundary potential if Dirichlet BCs are used. I think to keep things consistent those should be changed to use the z-potentials as well.